packages/vsce/__tests__/__unit__/doc/meta.program.test.ts

```diff
@@ -35,6 +37,18 @@ describe("Program Meta", () => {
     const label = ProgramMeta.getLabel(programMock);
     expect(label).toEqual(`MYPROG`);
   });
+  it("should return library", () => {
+    programMock.attributes.library = "MYLIB";
+    ProgramMeta!.getLibrary = jest.fn(() => programMock.attributes.library);
+    const label = ProgramMeta.getLibrary(programMock);
+    expect(label).toEqual(`MYLIB`);
+  });
+  it("should return libraryDsn", () => {
+    programMock.attributes.librarydsn = "MYLIBDSN";
+    ProgramMeta!.getLibraryDsn = jest.fn(() => programMock.attributes.librarydsn);
+    const label = ProgramMeta.getLibraryDsn(programMock);
+    expect(label).toEqual(`MYLIBDSN`);
+  });
```

Comment:
These don't seem like they're actually testing anything. You're...
1. setting the value
2. creating a new method, that returns the value
3. asserting the method you just defined does the thing you told it to do.

The fact that method doesn't exist on the meta file, we probably don't need to do this.